tests/unit/preprocessor/_time/test_time.py

```diff
@@ -1022,6 +1022,19 @@ def test_standardized_anomalies(period, standardize=True):
             )
 
 
+@pytest.mark.parametrize('period, reference', PARAMETERS)
+def test_anomalies_preserve_metadata(period, reference, standardize=False):
+    cube = make_map_data(number_years=2)
+    cube.var_name = "si"
+    cube.units = "m"
+    result = anomalies(cube, period, reference, standardize=standardize)
+    assert result.metadata == cube.metadata
+    for coord_cube, coord_res in zip(cube.coords(), result.coords()):
+        if coord_cube.has_bounds() and coord_res.has_bounds():
+            assert_array_equal(coord_cube.bounds, coord_res.bounds)
+        assert coord_cube == coord_res
```

Comment:
I'm not sure if this is needed, because you're testing that the metadata was preserved, right?

Reply:
the metadata object does not contain information on the coords, I was quite surprised by that too